app/assets/stylesheets/activities.css.scss

```diff
@@ -16,7 +16,7 @@ article {
 
 #hiking-card{
         height: 176px;
-        background: url('hiker.jpg') center / cover;
+        background: image-url('hiker.jpg') center / cover;
```

Comment:
Line should be indented 2 spaces, but was indented 8 spaces<br>Prefer double-quoted strings